src/full/Agda/Syntax/Translation/AbstractToConcrete.hs

```diff
@@ -909,7 +909,8 @@ instance ToConcrete A.Expr where
         bracket piBrackets
         $ do a' <- toConcreteCtx ctx a
              b' <- toConcreteTop b
-             let dom = setQuantity (getQuantity a') $ defaultArg $ addRel a' $ mkArg a'
+             -- XXX We set relevance to Relevant in arginfo because we instead use C.Dot or C.DoubleDot
```

Comment:
Does XXX stand for anything here?  If yes, please expand, otherwise please delete.

Reply:
Ah, that's a common code tag like `TODO` or `NOTE` meant to convey that this is non-intuitive stuff that is not necessarily buggy either. I guess it's not that widespread in the Agda codebase though, I can change it to something else like `NOTE`.